operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -0,0 +1,785 @@
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - S3
+    - MinIO
+    - Object Storage
+  labels:
+    operatorframework.io/arch.386: supported
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.amd64p32: supported
+    operatorframework.io/arch.arm: supported
+    operatorframework.io/arch.arm64: supported
+    operatorframework.io/arch.arm64be: supported
+    operatorframework.io/arch.armbe: supported
+    operatorframework.io/arch.loong64: supported
+    operatorframework.io/arch.mips: supported
+    operatorframework.io/arch.mips64: supported
+    operatorframework.io/arch.mips64le: supported
+    operatorframework.io/arch.mips64p32: supported
+    operatorframework.io/arch.mips64p32le: supported
+    operatorframework.io/arch.mipsle: supported
+    operatorframework.io/arch.ppc: supported
+    operatorframework.io/arch.ppc64: supported
+    operatorframework.io/arch.ppc64le: supported
+    operatorframework.io/arch.riscv: supported
+    operatorframework.io/arch.riscv64: supported
+    operatorframework.io/arch.s390: supported
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.sparc: supported
+    operatorframework.io/arch.sparc64: supported
+    operatorframework.io/arch.wasm: supported
+    operatorframework.io/os.aix: supported
+    operatorframework.io/os.android: supported
+    operatorframework.io/os.darwin: supported
+    operatorframework.io/os.dragonfly: supported
+    operatorframework.io/os.freebsd: supported
+    operatorframework.io/os.hurd: supported
+    operatorframework.io/os.illumos: supported
+    operatorframework.io/os.ios: supported
+    operatorframework.io/os.js: supported
+    operatorframework.io/os.linux: supported
+    operatorframework.io/os.nacl: supported
+    operatorframework.io/os.netbsd: supported
+    operatorframework.io/os.openbsd: supported
+    operatorframework.io/os.plan9: supported
+    operatorframework.io/os.solaris: supported
+    operatorframework.io/os.windows: supported
+    operatorframework.io/os.zos: supported
+  links:
+    - name: Website
+      url: https://min.io
+    - name: Support
+      url: https://subnet.min.io
+    - name: Github
+      url: https://github.com/minio/operator
+  maintainers:
+    - email: dev@min.io
+      name: MinIO Team
+  maturity: stable
+  provider:
+    name: MinIO Inc
+    url: https://min.io
+  relatedImages:
+    - image: quay.io/minio/operator@sha256:db47d598488f10daf6a4d26431ab2b65e24f3dffbae0edb76bc44f63c6daf500
+      name: console
+    - image: quay.io/minio/operator@sha256:db47d598488f10daf6a4d26431ab2b65e24f3dffbae0edb76bc44f63c6daf500
+      name: minio-operator
+    - image: quay.io/minio/minio@sha256:ea2c70cda70860c7d10f04ce1df640d3abb995784cb7aeedef8d4baa53a5a113
+      name: minio-ea2c70cda70860c7d10f04ce1df640d3abb995784cb7aeedef8d4baa53a5a113-annotation
+  version: 5.0.14
+  skips: minio-operator-rhmp.v5.0.13
```

Comment:
`skips` is an array and must be formatted properly.
```
skips:
   - minio-operator-rhmp.v5.0.13
```

Reply:
got it, on it